### PR TITLE
bump HF versions

### PR DESCRIPTION
**Pull request by darcys22**
Hardfork 17 will transition for all users already on 8.x.x

Hardfork 18 to be afterwards with ONS updates, Decommission reasons, btencoded uptime proofs etc

### CODE REVIEW

File: src/cryptonote_basic/hardfork.cpp [hunk 1]
```diff
@@ -75,6 +75,7 @@ static constexpr HardFork::Params mainnet_hard_forks[] =
   { network_version_15_lns,                 496969, 0, 1585105200 }, // 2020-03-25 14:00AEDT (03:00UTC)
   { network_version_16_pulse,               641111, 0, 1602464400 }, // 2020-10-12 12:00AEDT (01:00UTC)
   { network_version_17,                     641111 + BLOCKS_EXPECTED_IN_DAYS(180), 0, 1602464400 + 180*24*60*60 }, // HF16 + 180 days
+  { network_version_18,                     641111 + BLOCKS_EXPECTED_IN_DAYS(200), 0, 1602464400 + 200*24*60*60 }, // HF16 + 200 days
 };
```
Comment (darcys22) [inline src/cryptonote_basic/hardfork.cpp:79]: To confirm and finalise actual HF times

File: src/cryptonote_basic/hardfork.cpp [hunk 2]
```diff
@@ -75,6 +75,7 @@ static constexpr HardFork::Params mainnet_hard_forks[] =
   { network_version_15_lns,                 496969, 0, 1585105200 }, // 2020-03-25 14:00AEDT (03:00UTC)
   { network_version_16_pulse,               641111, 0, 1602464400 }, // 2020-10-12 12:00AEDT (01:00UTC)
   { network_version_17,                     641111 + BLOCKS_EXPECTED_IN_DAYS(180), 0, 1602464400 + 180*24*60*60 }, // HF16 + 180 days
+  { network_version_18,                     641111 + BLOCKS_EXPECTED_IN_DAYS(200), 0, 1602464400 + 200*24*60*60 }, // HF16 + 200 days
```
Comment (jagerman) [inline src/cryptonote_basic/hardfork.cpp:78]: Leave this one off for now to leave the mainnet HF unscheduled until we have a concrete date

File: src/cryptonote_basic/hardfork.cpp [hunk 3]
```diff
@@ -90,6 +91,7 @@ static constexpr HardFork::Params testnet_hard_forks[] =
   { network_version_15_lns,                 244777, 0, 1583940000 }, // 2020-03-11 15:20UTC
   { network_version_16_pulse,               382222, 0, 1600468200 }, // 2020-09-18 22:30UTC
   { network_version_17,                     447275, 0, 1608276840 }, // 2020-12-18 05:34UTC 
+  { network_version_18,                     447275, 0, 1608276840 }, // 2020-12-18 05:34UTC
```
Comment (jagerman) [inline src/cryptonote_basic/hardfork.cpp:94]: The current testnet syncs up to 501743, so let's set this to 501750.  Timestamp can be the current time to be lazy: `1616631051`

**Review by jagerman — APPROVED** [review]

File: src/cryptonote_basic/hardfork.cpp (hunk 1 above)
Comment (jagerman) [reply src/cryptonote_basic/hardfork.cpp]: Also when we do bump this there's no need to keep that BLOCKS_EXPECTED_IN_DAYS stuff for the new HF -- that was just there to be explicit that the fork was deliberately 180 days (for the block reward expiry).

**Pull request merged**